src/support/slack/commands/get-sites.js

```diff
@@ -79,19 +79,32 @@ function GetSitesCommand(context) {
     name: 'Get All Sites',
     description: 'Retrieves all known sites and includes the latest audit scores',
     phrases: PHRASES,
-    usageText: `${PHRASES.join(' or ')} [desktop|mobile|all] [live|non-live] [aem_edge|aem_cs|other];`,
+    usageText: `${PHRASES.join(' or ')} [desktop|mobile|all] [live|non-live] [aem_edge|aem_cs|other] [<IMSOrgId>];`,
   });
 
   const { dataAccess, log } = context;
-  const { Site } = dataAccess;
+  const { Organization, Site } = dataAccess;
+
+  async function fetchAndFormatSites(threadTs, filterStatus, psiStrategy, deliveryType, imsOrgId) {
+    let orgFilter = (s) => s.getOrganizationId() !== context.env.ORGANIZATION_ID_FRIENDS_FAMILY;
+
+    if (imsOrgId !== 'all') {
+      const org = await Organization.findByImsOrgId(imsOrgId);
+      const organizationId = org?.getId();
+      if (!hasText(organizationId)) {
+        return {
+          textSections: [{
+            text: `*No organization found in Spacecat DB with IMS Org ID: "${imsOrgId}"`,
+          }],
+          additionalBlocks: [],
+        };
+      }
+      orgFilter = (site) => site.getOrganizationId() === organizationId;
```

Comment:
for efficiency purposes we should not filter the sites, but use `organization.getSites()` instead...